### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by ghost** [Irautak/FingerFlex#3]
Hi,
  Your work is so wonderful! So, I'd like to try it, I download the dataset from the BCI Comp webseite (https://www.bbci.de/competition/iv/#download), but only three .mat files in it(sub1_comp.mat, sub2_comp.mat and sub3_comp.mat)，run your code prepare_data.ipynb, it dosen't work, could you tell me how to get sub1_testlabels.mat？

**Comment by oanhha9807** [comment]
Same question

**Comment by wpf535236337** [comment]
https://www.bbci.de/competition/iv/results/index.html#labels
![result](https://github.com/user-attachments/assets/550ad3d6-bfa8-4a34-a0ef-6da700e97e3d)

**Comment by Kei713** [comment]
hI, I read you article and the code, I'm confused, you use 400s of data provided by the website as the training set and 200s as the validation set, so what is the test set, I haven't seen the description of the test set, continue to use the 200s data as test data?

**Comment by Daniel-Lin-S** [comment]
> hI, I read you article and the code, I'm confused, you use 400s of data provided by the website as the training set and 200s as the validation set, so what is the test set, I haven't seen the description of the test set, continue to use the 200s data as test data?

As described in "training details" section of the original paper, the authors used the last 200s of data that was supposed to be test set in the competition for hyper-parameter tuning and validation. After properly performing train-validation-test split, I found a significant performance drop compared to the metrics reported in the original paper. For example, average Pearson correlation of subject 1 dropped to 0.52.
